_posts/10-03-01-Password-Hashing.md

```diff
@@ -12,8 +12,13 @@ It is important that you properly [_hash_][3] passwords before storing them. Pas
 way function performed against the user's password. This produces a fixed-length string that cannot be feasibly
 reversed. This means you can compare a hash against another to determine if they both came from the same source string,
 but you cannot determine the original string. If passwords are not hashed and your database is accessed by an
-unauthorized third-party, all user accounts are now compromised. Some users may (unfortunately) use the same password
-for other services. Therefore, it is important to take security seriously.
+unauthorized third-party, all user accounts are now compromised. 
+
+Passwords should also be individually [_salted_][5] by adding a random string to each password before hashing. This prevents dictionary attacks and the use of 'rainbow tables' (a reverse list of crytographic hashes for common passwords.)
```

Comment:
'rainbow tables' should have double-quotes (US quotes) instead of single-quotes (UK quotes), at least according to the "Contributor Syle Guide" in [CONTRIBUTING.md](https://github.com/codeguy/php-the-right-way/blob/gh-pages/CONTRIBUTING.md).